admin_manual/ai/app_context_chat.rst

```diff
@@ -11,15 +11,15 @@ Context Chat is an :ref:`assistant<ai-app-assistant>` feature that is implemente
 
 Together they provide the ContextChat text processing tasks accessible via the :ref:`Nextcloud Assistant app<ai-app-assistant>`.
 
-The *context_chat* and *context_chat_backend* apps run only open source models and do so entirely on-premises. Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.
+The *context_chat* and *context_chat_backend* apps will use the Free text to text task processing providers like OpenAI integration, LLM2, etc. and such a provider is required on a fresh install, or it can be configured to run open source models entirely on-premises. Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.
 
 This app supports input and output in languages other than English if the language model supports the language.
 
 Requirements
 ------------
 
-* Minimal Nextcloud version: 28
-* The *context_chat_backend* app is built as an External App and thus depends on AppAPI v2.3.0 and later
```

Comment:
I think we should be specific about versions

Reply:
If we're requiring nextcloud 30, then let's remove the note about "Below version 3" in the CPU requirements